From aea5268452b8cf31f01bd1051cbe4728a87ac35a Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Thu, 1 Oct 2020 16:39:00 -0500
Subject: [PATCH] Ensure libffmpeg.so is in the rpath of binaries

---
 BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/third_party/ffmpeg/BUILD.gn b/third_party/ffmpeg/BUILD.gn
index c01a320e69..f8282eb69f 100755
--- a/third_party/ffmpeg/BUILD.gn
+++ b/third_party/ffmpeg/BUILD.gn
@@ -451,6 +451,12 @@ if (is_component_ffmpeg) {
     if (is_mac && !is_component_build) {
       ldflags += [ "-Wl,-install_name,@rpath/libffmpeg.dylib" ]
     }
+
+    if (is_linux && !is_component_build) {
+      all_dependent_configs = [
+        "//build/config/gcc:rpath_for_built_shared_libraries",
+      ]
+    }
   }
 } else {
   source_set("ffmpeg") {
-- 
2.31.1

