From bd008f9daf231f7e797208060c3057b9cebb6f6a Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <git@refi64.dev>
Date: Tue, 16 Apr 2024 19:52:03 -0500
Subject: [PATCH] Skip formatting generated GN files

---
 chromium/scripts/generate_gn.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/ffmpeg/chromium/scripts/generate_gn.py b/third_party/ffmpeg/chromium/scripts/generate_gn.py
index 616ed6711a..9caf1b6b56 100755
--- a/third_party/ffmpeg/chromium/scripts/generate_gn.py
+++ b/third_party/ffmpeg/chromium/scripts/generate_gn.py
@@ -1054,7 +1054,7 @@ def main():
     with open(gn_file_name, 'w') as fd:
         WriteGn(fd, sets)
 
-    subprocess.run(['gn', 'format', gn_file_name])
+    # subprocess.run(['gn', 'format', gn_file_name])
 
 if __name__ == '__main__':
     main()
-- 
2.44.0

