From 77db05a7d1f64ab1d1bf2bc6146bcd965bbdcaa6 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Thu, 1 Oct 2020 16:39:00 -0500
Subject: [PATCH 4/4] Ensure libffmpeg.so is in the rpath of binaries

---
 third_party/ffmpeg/BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/third_party/ffmpeg/BUILD.gn b/third_party/ffmpeg/BUILD.gn
index cbe2fdd844..9429121215 100755
--- a/third_party/ffmpeg/BUILD.gn
+++ b/third_party/ffmpeg/BUILD.gn
@@ -462,6 +462,12 @@ if (is_component_ffmpeg) {
     if (is_mac && !is_component_build) {
       ldflags += [ "-Wl,-install_name,@rpath/libffmpeg.dylib" ]
     }
+
+    if (is_linux && !is_component_build) {
+      all_dependent_configs = [
+        "//build/config/gcc:rpath_for_built_shared_libraries",
+      ]
+    }
   }
 } else {
   source_set("ffmpeg") {
-- 
2.31.1

