From 368f309b95da13e0385492d489f7b9953dbc8f11 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Wed, 2 Jun 2021 20:32:22 -0500
Subject: [PATCH 25/25] build: Remove arm64 tool prefix

---
 build/toolchain/linux/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/toolchain/linux/BUILD.gn b/build/toolchain/linux/BUILD.gn
index 39f963f4c2755..ee59b07bb08ba 100644
--- a/build/toolchain/linux/BUILD.gn
+++ b/build/toolchain/linux/BUILD.gn
@@ -23,7 +23,7 @@ clang_toolchain("clang_arm") {
 }
 
 clang_toolchain("clang_arm64") {
-  toolprefix = "aarch64-linux-gnu-"
+  toolprefix = ""
   toolchain_args = {
     current_cpu = "arm64"
     current_os = "linux"
-- 
2.31.1

