From 717f11114c1f71822a0a9e079037dba066682606 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Wed, 31 Aug 2022 10:09:01 -0500
Subject: [PATCH] Clang build script: Don't build against the sysroot

---
 tools/clang/scripts/build.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/tools/clang/scripts/build.py b/tools/clang/scripts/build.py
index fe6de40d8bc96..338650c6c02eb 100755
--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -785,7 +785,7 @@ def main():
     if sys.platform.startswith('linux'):
       base_cmake_args += [ '-DLLVM_STATIC_LINK_CXX_STDLIB=ON' ]
 
-  if sys.platform.startswith('linux'):
+  if sys.platform.startswith('linux') and False:
     sysroot_amd64 = DownloadDebianSysroot('amd64')
     sysroot_i386 = DownloadDebianSysroot('i386')
     sysroot_arm = DownloadDebianSysroot('arm')
@@ -1069,7 +1069,7 @@ def main():
   if sys.platform.startswith('linux'):
     runtimes_triples_args['i386-unknown-linux-gnu'] = {
         "args": [
-            'CMAKE_SYSROOT=%s' % sysroot_i386,
+            # 'CMAKE_SYSROOT=%s' % sysroot_i386,
             # TODO(https://crbug.com/1374690): pass proper flags to i386 tests so they compile correctly
             'LLVM_INCLUDE_TESTS=OFF',
         ],
@@ -1080,7 +1080,7 @@ def main():
     }
     runtimes_triples_args['x86_64-unknown-linux-gnu'] = {
         "args": [
-            'CMAKE_SYSROOT=%s' % sysroot_amd64,
+            # 'CMAKE_SYSROOT=%s' % sysroot_amd64,
         ],
         "profile": True,
         "sanitizers": True,
@@ -1094,7 +1094,7 @@ def main():
     # Maybe it should work for builtins too?
     runtimes_triples_args['armv7-unknown-linux-gnueabihf'] = {
         "args": [
-            'CMAKE_SYSROOT=%s' % sysroot_arm,
+            # 'CMAKE_SYSROOT=%s' % sysroot_arm,
             # Can't run tests on x86 host.
             'LLVM_INCLUDE_TESTS=OFF',
         ],
@@ -1105,7 +1105,7 @@ def main():
     }
     runtimes_triples_args['aarch64-unknown-linux-gnu'] = {
         "args": [
-            'CMAKE_SYSROOT=%s' % sysroot_arm64,
+            # 'CMAKE_SYSROOT=%s' % sysroot_arm64,
             # Can't run tests on x86 host.
             'LLVM_INCLUDE_TESTS=OFF',
         ],
-- 
2.41.0

