From b6e23330fb16512992abb329da3400a4eccab6d9 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Mon, 19 Dec 2022 19:58:16 -0600
Subject: [PATCH] Ignore useless warnings from the Asahi driver

---
 ui/gl/gl_display.cc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/ui/gl/gl_display.cc b/ui/gl/gl_display.cc
index 846d968fec6b5..fae344fd3e4e7 100644
--- a/ui/gl/gl_display.cc
+++ b/ui/gl/gl_display.cc
@@ -485,6 +485,12 @@ void EGLAPIENTRY LogEGLDebugMessage(EGLenum error,
                                     EGLLabelKHR thread_label,
                                     EGLLabelKHR object_label,
                                     const char* message) {
+  if (message == std::string("_eglClientWaitSyncCommon") ||
+      message == std::string("_eglDestroySync")) {
+    // Unimplemented on the Asahi driver, so just ignore.
+    return;
+  }
+
   std::string formatted_message = std::string("EGL Driver message (") +
                                   GetDebugMessageTypeString(message_type) +
                                   ") " + command + ": " + message;
-- 
2.39.0

