From 94db1f9f2e39100e5323d5d39e642c27886a1254 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Mon, 19 Dec 2022 19:58:16 -0600
Subject: [PATCH] Ignore useless warnings from the Asahi driver

---
 ui/gl/gl_display.cc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/ui/gl/gl_display.cc b/ui/gl/gl_display.cc
index 038387570aa58..4069ae688939e 100644
--- a/ui/gl/gl_display.cc
+++ b/ui/gl/gl_display.cc
@@ -479,6 +479,12 @@ void EGLAPIENTRY LogEGLDebugMessage(EGLenum error,
                                     EGLLabelKHR thread_label,
                                     EGLLabelKHR object_label,
                                     const char* message) {
+  if (message == std::string("_eglClientWaitSyncCommon") ||
+      message == std::string("_eglDestroySync")) {
+    // Unimplemented on the Asahi driver, so just ignore.
+    return;
+  }
+
   std::string formatted_message = std::string("EGL Driver message (") +
                                   GetDebugMessageTypeString(message_type) +
                                   ") " + command + ": " + message;
-- 
2.39.1

